cranelift/codegen/src/isa/x64/lower.rs

```diff
@@ -1415,6 +1446,11 @@ fn lower_insn_to_regs<C: LowerCtx<I = Inst>>(
             ));
         }
 
+        Opcode::RawBitcast => {
+            // Emit nothing; a raw_bitcast is just a mechanism for correcting the type of V128
+            // values (see https://github.com/bytecodealliance/wasmtime/issues/1147).
```

Comment:
Yeah, as Chris noted, we need a way to "connect" the input and output registers, so we need to generate a move instruction here; again, it won't necessarily materialize as a move instruction.